webstorage/idlharness.html

```diff
@@ -73,8 +69,11 @@ <h1>Web Storage IDL tests</h1>
   idl_array.add_objects({Storage: ["window.localStorage"]});
 
   idl_array.test();
-}, 'webstorage interfaces');
+}
 
+promise_test(function () {
+  return Promise.all(['/interfaces/html.idl', '/interfaces/dom.idl'].map(fetchText)).then(doTest);
```

Comment:
How does this work? Did you mean `do_test` instead of `doTest`?

Reply:
You saw nothing.